managing_providers/cloud_providers/_topics/enabling_cu_data_from_ibm_power_vc_providers.md

```diff
@@ -0,0 +1,27 @@
+## Enabling Capacity and Utilization data from IBM PowerVC Providers
+
+Complete the following procedure to enable Capacity and Utilization data for an
```

Comment:
It might be helpful to mention that these instructions are in additional to enabling C&U in general. Some like the following:
```suggestion
In addition to enabling general [Capacity and Utilization Collection](../../general_configuration/index.html#capacity-and-utilization-collection) complete the following procedure to enable Capacity and Utilization data for an
```
disclaimer: not sure if that relative link is correct 😕

Reply:
Thanks! That link worked for me. I will update the PR with your suggestion.